.cicd/platforms/ubuntu22.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:jammy
+FROM ubuntu:kinetic
```

Comment:
Did you mean to change this? If you really want to test 22.10 I'd suggest creating a new platform definition instead of replacing 22.04: we claim support for 22.04 so we need to make sure it continues to work.

Reply:
I thought we support latest 22.xx, not latest 22.04. Since 22.10 is not LTS it make sense, I'll switch it back.